backend/routes/SponsorshipFund.js

```diff
@@ -0,0 +1,69 @@
+const router = require("express").Router();
+let SponsorshipFund = require("../models/SponsorshipFund.model");
+
+router.route("/").get((_, res) => {
+  SponsorshipFund.find()
+    .then((SponsorshipFund) => res.json(SponsorshipFund))
+    .catch((err) => res.status(400).json(err));
+});
+
+router.route("/").post((req, res) => {
+  if (!req.body) {
```

Comment:
req.body always exists, if you don't pass it in it will be set to {} by default by express. Check if it is the empty object instead